clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/PartitionConsumptionState.java

```diff
@@ -187,6 +187,16 @@ public class PartitionConsumptionState {
    */
   private Map<String, Long> latestProcessedUpstreamRTOffsetMap;
 
+  private boolean restartIngestion;
+
+  public boolean isRestartIngestion() {
```

Comment:
Could you leave a JavaDoc on this function, to explain what the state of the system is when this returns true or false?

Also, `isRestartIngestion` sounds slightly weird to me. Perhaps `isIngestionRestarted`? Or maybe some other wording entirely, if that doesn't sound good... IDK.

Reply:
Makes sense. Added.